test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -995,9 +990,8 @@ class bcGeneralTestsFixture
 	public:
 	bcGeneralTestsFixture()
 	{
-		//general tests are filled from state tests
 		//skip this test suite if not run with --all flag (cases are already tested in state tests)
-		if (test::Options::get().filltests || !test::Options::get().all)
```

Comment:
Why does the behavior change here?

Reply:
You could now run dynamically generated from state tests blockchain test fillers as a test suite.
./testeth -t BCGeneralStateTests -- --filltests   
but only if you have --all flag specified.

Reply:
Why do you require `--all` here?  What is the reason for this sudden change?

Reply:
because this is a hive test suite

Reply:
This is a hive test suite before and after this PR.  That did not change.  I'm asking for the reason of the change.

Reply:
ok. now you could fill it as a test suite. before it was done only with --fillchain

Reply:
When I "fill it as a test suite", what kind of options do I use?

Reply:
--filltests  to fill.  it will work as a blockchain test suite.
it is just that fillers are generated dynamically.

Reply:
Are there two ways to do it?  Can we drop one of these?

Reply:
I prefer not to do it. It saves time.

Reply:
Why do you require `--all`?